boards/itsybitsy_m0/Cargo.toml

```diff
@@ -51,19 +78,57 @@ chip = "ATSAMD21G18A"
 name = "blinky_basic"
 
 [[example]]
-name = "dotstar"
-required-features = ["unproven"]
+name = "timers"
+
+[[example]]
+name = "pwm"
+
+[[example]]
+name = "adc"
+
+[[example]]
+name = "ssd1306_graphicsmode_128x64_i2c"
+
+[[example]]
+name = "ssd1306_terminalmode_128x64_i2c"
 
 [[example]]
-name = "i2c_ssd1306"
+name = "ssd1306_graphicsmode_128x64_spi"
 
 [[example]]
-name = "spi_ssd1306"
+name = "ssd1306_terminalmode_128x64_spi"
 
 [[example]]
 name = "usb_echo"
 required-features = ["usb"]
 
 [[example]]
 name = "twitching_usb_mouse"
```

Comment:
Does this example need the `usb` feature?

Reply:
which one do you mean?

Reply:
The `twitching_usb_mouse` example. Sorry for the confusion!

Reply:
Ok thanks, done in fd8f90e08942185cd4f4f6f1f50cf9ac5e5dddc2